public/js/dashboard.js

```diff
@@ -13,33 +14,56 @@ const findAncestor = (el, cls) => {
   return el;
 };
 
-const resendEmail = async(e) => {
-  const resendLink = e.target;
-  const parentDiv = findAncestor(resendLink, "e-toggle-info-wrapper");
-  const emailToSend = parentDiv.querySelector("span.unverified-e-address").innerText;
-
-  JSON.stringify(emailToSend);
-
-  await fetch("/user/resend-email", {
+const sendForm = async(action, formBody={}) => {
+  const response = await fetch(`/user/${action}`, {
     headers: {
       "Content-Type": "application/json; charset=utf-8",
     },
     mode: "cors",
     method: "POST",
-    body: JSON.stringify({ "email": emailToSend }),
+    body: JSON.stringify(formBody),
   });
+
+  return await response.json();
+};
+
+const sendCommunicationOption = async(e) => {
+  const radioButton = e.target;
+  const formAction = radioButton.dataset.formAction;
+  const option = radioButton.dataset.commOption;
+  sendForm(formAction, { communicationOption: option })
+    .then(data => {}) /*decide what to do with data */
+    .catch(e => {})/* decide how to handle errors */;
+  };
+
+const removeOrResendEmail = async(e) => {
```

Comment:
function declaration

Reply:
Also, this function makes me realize we're going to have to have `sendForm` include CSRF values. We'll address that when we implement the back-end routes + controllers.